pages/404.tsx

```diff
@@ -0,0 +1,17 @@
+import { useRouter } from "next/router";
+import { LandingNav, LandingFooter } from "@/Components/components";
+
+const NotFoundPage = () => {
+  return (
+    <>
+      <LandingNav pageNotFound={true} />
+      <div className="flex items-center justify-center h-[500px] flex-col m-8 text-2xl">
```

Comment:
Please do not use hard-coded values `h-[500px]`, you can use `h-1/2` or `h-1/4` or other pre-defined classes of tailwind.

Reply:
Done